CONTRIBUTING.md

```diff
@@ -130,3 +130,23 @@ import (
 	"github.com/grafana/loki/pkg/logql"
 )
 ```
+
+## Contribute to documentation
+
+We're glad you're here to help make our technical documentation even better for Loki users.
+
+The Grafana docs team has created a [Writer's Toolkit](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-documentation/) that includes information about how we write docs, a [Style Guide](https://grafana.com/docs/writers-toolkit/style-guide/), and templates to help you contribute to the Loki documentation. 
+
+The Loki documentation is written using the CommonMark flavor of markdown, including some extended features. For more information about markdown, you can see the [CommonMark specification](https://spec.commonmark.org/), and a [quick reference guide](https://commonmark.org/help/) for CommonMark.
+
+Loki uses the static site generator [Hugo](https://gohugo.io/) to generate the documentation. Loki uses a continuous integration (CI) action to sync documentation to the [Grafana website](https://grafana.com/docs/loki/latest). The CI is triggered on every merge to main in the `docs` subfolder.
+
+You can preview the documentation locally after installing both [Docker](https://www.docker.com/) and Hugo.  
```

Comment:
Hugo is already included in the Docker image for building the documentation, so you don't need to install it separately.

Reply:
Instead of Docker, you could also install [Podman](https://podman.io).